shared/router-v2/util.tsx

```diff
@@ -13,13 +14,15 @@ export const getActiveIndex = (navState: NavState): number => {
   return navState.index
 }
 // Get active key inside any subnavigator so navigation closures are
-export const getActiveKey = (navState: NavState): string => {
-  if (!navState.routes) {
-    return navState.key
+export const getActiveKey = memoize(
```

Comment:
does this memoizing work at all. this thing is called recursively so you're blowing it away right

Reply:
Good catch, fixed.